client/src/featureform/register.py

```diff
@@ -379,6 +387,28 @@ def average_user_transaction(df):                           # Sources can be man
             properties=properties,
         )
 
+    def _validate_file_scheme(self, store_type: str, file_path: str) -> (bool, str):
```

Comment:
it might make sense to move this function on to the enum itself.

```suggestion
    class FilePrefix(Enum):
    S3 = "s3://"
    S3A = "s3a://"
    HDFS = "hdfs://"
    GCS = "gs://"
    AZURE = "abfss://"
    
    def validate(self):
       ...
```